xpdan/data_reduction.py

```diff
@@ -299,7 +324,10 @@ def integrate_and_save(headers, dark_sub_bool=True,
                                    [chi_fn_Q, chi_fn_2th],
                                    [header_rv_list_Q, header_rv_list_2theta]):
                 print("INFO: save chi file: {}".format(fn))
-                rv = ai.integrate1d(img, npt, filename=fn, mask=~mask,
+                if mask is not None:
+                    # flip before handing in mask
+                    mask = ~mask
```

Comment:
I would seriously advise against this, as whatever happens we will no longer be able to  load our masks into our own functions.